src/styles/transitions.scss

```diff
@@ -0,0 +1,46 @@
+.item-enter {
+  max-height: 0;
+}
+
+.item-enter-active {
+  overflow: hidden;
+  max-height: 58px;
+  transition: max-height 0.3s ease-in-out;
```

Comment:
It is recommended to use variables for transition durations to keep the code consistent and easy to maintain. Consider defining a variable for the transition duration (e.g., `$transition-duration: 0.3s;`) and using it instead of the hardcoded value.